lib/resty/limit/conn.lua

```diff
@@ -23,6 +23,29 @@ local mt = {
 }
 
 
+local function incr_or_init(key, dict)
```

Comment:
@es Nope. This race condition should be minor. And I don't like using the goto feature in your PR. The Lua 5.2 feature can be disabled in the LuaJIT builds (and which is the default) and we shouldn't rely on it.